README.md

```diff
@@ -38,6 +38,11 @@ Works with Digoo DG-R8H 433MHz Wireless Digital Hygrometer Thermometer Weather S
 
 This plugin should work with a broad number of OOK based devices such as power plugs and remotes
 
+### fanju 
+
```

Comment:
Please add a sentence explaining what it is, in the reason to link these 2 websites